src/routes/docs/tutorials/pagination/step-6/+page.markdoc

```diff
@@ -101,4 +101,6 @@ export function Tasks() {
     </section>
   );
 }
-```
\ No newline at end of file
+```
+
+In the next two steps, we will explore how to use offset or cursor pagination in our application.
```

Comment:
should I put this in a info box with warning type?

Reply:
Nah, it's fine.